src/components/Joke.test.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { render, waitFor } from "@testing-library/react";
+
+import Joke from "./Joke";
+
+describe("Joke Component", () => {
+  it("should load only setup on intial render", () => {
+    const { getByText } = render(
+      <Joke setup="setup test" punchline="punchline test" />
+    );
+    const jokeElement = getByText("setup test");
+    expect(jokeElement).toBeTruthy();
```

Comment:
`toBeInTheDocument` might be a clearer assertion here

Reply:
I agree. Done!